source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -963,6 +963,115 @@ namespace RendererQml
 
 		return uiRectangle;
 	}
+
+	std::shared_ptr<QmlTag> RendererQml::AdaptiveCardQmlRenderer::GetNewColumn(std::shared_ptr<AdaptiveCards::Container> container, std::shared_ptr<AdaptiveRenderContext> context)
+	{
+		const auto margin = std::to_string(context->GetConfig()->GetSpacing().paddingSpacing);
+		const auto spacing = Utils::GetSpacing(context->GetConfig()->GetSpacing(), container->GetSpacing());
+
+		std::shared_ptr<QmlTag> uiColumn = std::make_shared<QmlTag>("Column");
+		if (container->GetPadding()) {
+			uiColumn->Property("Layout.margins", margin);
+		}
+		uiColumn->Property("Layout.fillWidth", "true");
+		uiColumn->Property("spacing", std::to_string(spacing));
+		if (container->GetVerticalContentAlignment() == AdaptiveCards::VerticalContentAlignment::Top) {
+			uiColumn->Property("Layout.alignment", "Qt.AlignTop");
+		}
+		else if (container->GetVerticalContentAlignment() == AdaptiveCards::VerticalContentAlignment::Center) {
+			uiColumn->Property("Layout.alignment", "Qt.AlignVCenter");
+		}
+		else {
+			uiColumn->Property("Layout.alignment", "Qt.AlignBottom");
+		}
+
+		return uiColumn;
+	}
+
+	std::shared_ptr<QmlTag> RendererQml::AdaptiveCardQmlRenderer::ContainerRender(std::shared_ptr<AdaptiveCards::Container> container, std::shared_ptr<AdaptiveRenderContext> context)
+	{
+		const auto margin = std::to_string(context->GetConfig()->GetSpacing().paddingSpacing);
+		const auto spacing = Utils::GetSpacing(context->GetConfig()->GetSpacing(), container->GetSpacing());
+		const auto id = (!container->GetId().empty() ? container->GetId() : "container_"+std::to_string(rand()));
```

Comment:
Let's maintain a counter instead for generating a random number.. Please check with @mbaradi